test/fixtures/mocks/express_request.js

```diff
@@ -9,12 +10,15 @@ module.exports = function (body) {
     query: {
       title: 'foo-bar-baz',
       unSupportedKey: 'something',
-      include: 'books'
+      include: 'book,chapter'
     },
     body: {
       mock: {
-        key: 'value'
+        key: 'vaulue'
       }
+    },
+    headers: {
+      'content-type': 'application/vnd.api+json'
```

Comment:
2nd paragraph here: http://jsonapi.org/format/#crud
This is not a requirement under read.

Now the server must always RESPOND with the header, even in read. But it only requires it in the request when there's content to have a type.